content/docs/03-ai-sdk-core/15-tools-and-tool-calling.mdx

```diff
@@ -124,11 +122,13 @@ const result = await streamText({
   prompt: 'What is the weather in San Francisco?',
 });
 
-for await (const textPart of result.textStream) {
+for await (const textPart of textStream) {
   process.stdout.write(textPart);
 }
 ```
 
+Similar to `generateText`, you can access intermediate tool calls and results using the `steps` property in the result object.
```

Comment:
smol nit (could be overthinking) wonder if we need to say result object given you are destructuring so there is no "result"

Reply:
oh yeah hmm. i guess we can leave it for now